### PR TITLE
Miscellaneous updates

### PR DESCRIPTION
**Pull request by shaynevanasperen**

### CODE REVIEW

**Comment by JosephWoodward** [comment]
Have you been able to use this anywhere at work yet?

File: samples/Samples/Samples.csproj [hunk 1]
```diff
@@ -2,16 +2,17 @@
...
+		<NoWarn>$(NoWarn);CA2007;CA1819;CA1054;CA2234</NoWarn>
 	</PropertyGroup>
 
 	<ItemGroup>
 		<Content Remove="wwwroot/albums.json" />
 	</ItemGroup>
 	
 	<ItemGroup>
-		<PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.4" />
+		<PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.6" />
```
Comment (slang25) [inline samples/Samples/Samples.csproj]: The `Version` should be omitted for `Microsoft.AspNetCore.App`

File: src/Magneto/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -1 +0,0 @@
-﻿[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("Magneto.Tests")]
```
Comment (slang25) [inline src/Magneto/Properties/AssemblyInfo.cs]: I presume signing the tests was problematic

File: test/Magneto.Tests/Core/CachedQueryExtensions.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+		{
+			return cachedQuery.getStateProperty<string>("CacheKey");
+		}
+
+		public static TCachedResult GetCachedResult<TContext, TCacheEntryOptions, TCachedResult>(this CachedQuery<TContext, TCacheEntryOptions, TCachedResult> cachedQuery)
+		{
+			return cachedQuery.getStateProperty<TCachedResult>("CachedResult");
+		}
+
+		static T getStateProperty<T>(this object cachedQuery, string name)
```
Comment (slang25) [inline test/Magneto.Tests/Core/CachedQueryExtensions.cs]: camelCasing looks odd to me here

Comment (shaynevanasperen) [reply test/Magneto.Tests/Core/CachedQueryExtensions.cs]: This is the naming convention used in this project (private methods are camelCased).

**Comment by shaynevanasperen** [comment]
@JosephWoodward Yes, it's been in production code at Just Eat for over a year already!

**Pull request merged**